rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.desired_balance/10_basic.yml

```diff
@@ -17,6 +17,7 @@ setup:
   - gte: { stats.computation_converged_index: 0 }
   - gte: { stats.computation_time_in_millis: 0 }
   - gte: { stats.reconciliation_time_in_millis: 0 }
+  - is_true: 'cluster_balance_stats'
```

Comment:
```yml
---
"Test cluster_balance_stats":
  
  - setup:
    - skip:
      version: " - 8.6.99"

  - do:
      _internal.get_desired_balance: { }

  - is_true: 'cluster_balance_stats'
```

Reply:
For completeness, you could alternatively use the `node_selector` feature to specify that the request must go to a node of a particular version. Depends whether we expect this to work in a mixed-version cluster or not. In this case I'd go with a separate test as Artem suggests, because we don't care about the mixed-version case.

Reply:
Thanks all